webapp/packages/plugin-connections/src/ConnectionForm/Options/ConnectionOptionsTabService.ts

```diff
@@ -80,20 +80,6 @@ export class ConnectionOptionsTabService extends Bootstrap {
 
   load(): void {}
 
-  isProjectShared(state: IConnectionFormState): boolean {
-    if (state.projectId === null) {
-      return false;
-    }
-
-    const project = this.projectInfoResource.get(state.projectId);
-
-    if (!project) {
-      return false;
-    }
-
-    return isSharedProject(project);
-  }
-
```

Comment:
seems like it should be reverted

Reply:
moved it to the `projectInfoResource` cause it is needed everywhere